server/templates/static/faq.html

```diff
@@ -180,6 +180,26 @@ <h1>Frequently Asked Questions </h1>
     uptime.
   </p>
 
+  <p>
+    <b>Q: How do I cite datacommons.org?</b><br>
+    To cite charts and tools on this site, please use the following format.
+  </p>
+
+    <blockquote>
+    Data Commons {{ current_year }}, <i>Data Commons</i>, viewed {{ current_date }}, &#60;https://datacommons.org/&#62;.
+    </blockquote>
+
+  <p>
+    If citing data from a particular dataset, e.g. CDC Places, then use:
+  </p>
+
+    <blockquote>
+    Data Commons {{ current_year }}, CDC Places, electronic dataset, <i>Data Commons</i>, viewed {{ current_date }}, &#60;https://datacommons.org/&#62;.
```

Comment:
remove the trailing slash? It will redirect to get rid of it anyway

Reply:
done!